scipy/integrate/tests/test_quadrature.py

```diff
@@ -469,7 +469,7 @@ def func(x):
         assert_allclose(res.integral, ref, 1e-2)
 
 
-def cumulative_simpson_nd_reference(y, *, x=None, dx=None, initial=None, axis=-1):
+def cumulative_simpson_nd_ref(y, *, x=None, dx=None, initial=None, axis=-1):
```

Comment:
Did you just change the function name for the sole purpose of reducing the line length in the one case it is used below?

Reply:
Yes although in hindsight I don't no why because it makes no difference due to the formatting I finally settled on

Reply:
reverted